src/Profile.js

```diff
@@ -4,10 +4,92 @@ const Profile = () => {
   return (
     <div className="home">
       <div id="content">
-        <div className="row justify-content-md-center">
-          <h3>Profile Page</h3>
+        <div className="profile-edit">
+          <img
+            id="profile-pic"
+            alt="profile"
+            src={
+              "https://www.business2community.com/wp-content/uploads/2017/08/blank-profile-picture-973460_640.png"
+            }
+          />
+          <div className="edit-box">
+            <img
+              alt="edit"
+              id="edit-icon"
+              src="https://img.icons8.com/ios/50/000000/edit--v1.png"
+            />
+          </div>
+        </div>
+        <h1 id="profile-name">William Dunn</h1>
+        <h id="profile-username">@willyd23</h>
+        <div id="profile" className="about-me">
+          <h1>About Me</h1>
+          <div className="row">
+            <div className="col">
+              <p>Major: Computer Science</p>
+              <p>Graduation Term: Spring 2022</p>
+            </div>
+            <div className="edit-box2">
+              <img
+                alt="edit"
+                id="edit-icon"
+                src="https://img.icons8.com/ios/50/000000/edit--v1.png"
+              />
+            </div>
+          </div>
+        </div>
+        <div id="profile">
+          <h1>Discussion Interests</h1>
+          <div class="interest">
```

Comment:
this would be better as an array of interests (as strings) that you map through. and then when they click the edit button, we could have a boolean useState variable that changes to editing mode. in editing mode they should have a button next to the interest that removes it (a.k.a filter it out of the array). 

all of variables should be done with useState so the page updates properly.